ui/src/shared/containers/SetOrg.tsx

```diff
@@ -51,7 +51,11 @@ const SetOrg: FunctionComponent<Props> = ({
     }
 
     // else default to first org
-    router.push(`/orgs/${orgs[0].id}`)
+    if (orgs && orgs.length > 0) {
+      router.push(`/orgs/${orgs[0].id}`)
+    }
+    // if no orgs exist are returned, create one
+    router.push(`/orgs/new`)
```

Comment:
Should this statement be inside an else? If we have orgs, it will push the first org and then also create a /orgs/new

Is that what is intended?

Reply:
You're totally right. Good catch!